src/build-task/AugurkCLI/cli.ts

```diff
@@ -45,7 +46,8 @@ export function buildBaseToolRunner(command: string): ToolRunner {
              .arg(command)
              .arg(["--url", augurkUrl])
              .arg(["--productName", productName])
-             .argIf(groupName, ["--groupName", groupName]);
+             .argIf(groupName, ["--groupName", groupName])
+             .argIf(additionalArguments, additionalArguments);
```

Comment:
Not sure if this will work when there are multiple arguments provided. The toolrunner library is kinda picky about those things, especially since it will have to translate between different OS'es.

Reply:
So I tried this out, but it looks like we need to split out the additional arguments and pass them as an array for this to work.

Reply:
Mhm that's tricky, how would we differentiate between boolean flags like --useIntegratedSecurity and value flags with actual values? We would probably need to make the plugin aware of every possible cmdline flag to do so? If so, we might as well make every flag available as an actual field in the task instead of as additional arguments. What do you think

Reply:
I guess we could do something like [this](https://github.com/microsoft/azure-pipelines-tasks/blob/1d75fa8f66aa1cf7a9cb62946939f30f087b2969/Tasks/Common/webdeployment-common/deployusingmsdeploy.ts#L105) to handle those cases, although I agree it's not pretty. 

Then again, just adding all possible options to the task UI isn't very user friendly either and will just clutter the UI. The task already has quite a few options to set as it is.

Reply:
I believe it's possible to create different sections that are default collapsed/expanded so we could divide the options into appropriate sections and/or add every currently unlisted option under the Advanced collapsed list so by default users won't be bothered by it.

Reply:
We could do that yes. But I still feel that there's a need to be able to pass additional arguments for scenario's where the CLI has been updated to support more options but we haven't yet found the time to add them to this extension. So perhaps we could add the new options you need right now to that new (by default collapsed) group and add the additionalArguments in there as well.